src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const trList = document.querySelectorAll('tbody tr');
+const thList = document.querySelector('thead tr');
+const tbody = document.querySelector('tbody');
+
+const cellsArray = [];
+
+for (const cell of trList) {
+  cellsArray.push({
+    'Name': cell.children[0].textContent,
+    'Position': cell.children[1].textContent,
+    'Age': cell.children[2].textContent,
+    'Salary': +cell.children[3].textContent.replace('$', '').replace(/,/gi, ''),
```

Comment:
u can connect 2 replace in one
```suggestion
    'Salary': +cell.children[3].textContent.replace(/[$,]/g, ''),
```

Reply:
Good proposal. 
Merge them